modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -150,8 +150,6 @@ export default class GPUGridAggregator {
       // cache weights and position data to process when data is not changed
       weights: null,
       gridPositions: null,
-      positionsBuffer: null,
-      positions64xyLowBuffer: null,
```

Comment:
@1chandu I'm pretty sure these are no longer used. They are still mentioned in the GPUGridAggregator documentation though.

Reply:
Yes, ok to remove,  code related to building/using these buffers is already removed in #3777.